### DISCUSSION TITLE
Unable to upload photo for verification

### DISCUSSION THREAD

**Discussion by My-little-git** [community/community#111145] [GitHub Education]
I can not attach a photo of the document to confirm the status because this document is large and necessary information is on different pages of A4. Plus, there’s a translation nearby, and that’s four pages. Only one photo can be uploaded for confirmation and with a very low resolution of about 1000 px per 1000 px. You also need to upload a photo taken from the phone or from the laptop camera, and on the phone for some reason to download the photo can not turn on the rear cameras, no matter how I try. So how do I upload one photo to a front-facing camera where I can see four A4 sheets in 1000 resolution?

**Comment by LintangDev** [comment]
Same Problem as well here 🙌

**Comment by mishmanners** [comment]
Try positioning all the documents in view of the camera and click to take the photo. This will ensure the system captures the information and the team can reach out directly to you if they need to obtain a higher resolution copy.

**Reply by Todo56** [reply]
That is basically impossible, I had trouble uploading a fully readable A4 page, had to cut it in half. The system compresses the image a lot, too much in my opinion.

**Reply by nihilisticzebra** [reply]
exactly lol all my uploads looks like they were taken 30 years ago

**Comment by Xndworld** [comment]
Same problem
